### PR TITLE
[bugfix] Fixed in-set operator configuration

### PR DESCRIPTION
**Pull request by erheme318**
Ref: https://github.com/qandidate-labs/qandidate-toggle-bundle/issues/66

I've got the following error when defining `in-set` operator with
`values` as an array:

```
Expected scalar, but got array
```

Therefore, updated configuration in order to fix the problem.

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -63,7 +63,13 @@ public function getConfigTreeBuilder()
                                         ->scalarNode('name')->end()
                                         ->scalarNode('key')->end()
                                         ->arrayNode('operator')
-                                            ->prototype('scalar')
```
Comment (erheme318) [inline DependencyInjection/Configuration.php]: We can also have same result by changing `scalar` to `variable`.

**Comment by erheme318** [comment]
I cannot see CI result in the Pull Request.
Tests passed.
https://travis-ci.org/qandidate-labs/qandidate-toggle-bundle/builds/463299295

**Comment by erheme318** [comment]
@asm89 @wjzijderveld @othillo please review

**Comment by othillo** [comment]
Ideally we'd have only 1 key to specify values (https://github.com/qandidate-labs/qandidate-toggle/issues/50) but that would be a BC break.

For now I'm OK with prototyping the operator node as 'variable'

File: DependencyInjection/Configuration.php (hunk 1 above)
Comment (othillo) [reply DependencyInjection/Configuration.php]: I prefer changing this to 'variable' over specifying (optional) nodes for 'value' and 'values. Can you change this?

Comment (erheme318) [reply DependencyInjection/Configuration.php]: Of course

Comment (erheme318) [reply DependencyInjection/Configuration.php]: done

**Comment by wjzijderveld** [comment]
Thanks @erheme318 !

**Pull request merged**